### ISSUE TITLE
2.8.2 Typings/Type Definitions error for Graphics

### ISSUE THREAD

**Issue by dolanmiu** [photonstorm/phaser-ce#284]
* An error in the TypeScript defs

I get these errors:

```
src/prefabs/water/water.ts(26,18): error TS2339: Property 'beginFill' does not exist on type 'Graphics'.
src/prefabs/water/water.ts(28,18): error TS2339: Property 'drawPolygon' does not exist on type 'Graphics'.
src/states/main.ts(60,23): error TS2339: Property 'clear' does not exist on type 'Graphics'.
src/states/main.ts(62,23): error TS2339: Property 'endFill' does not exist on type 'Graphics'.
```

I fixed it by installing `phaser-ce@2.8.1`

**Comment by samme** [comment]
Could you test c975bc9?

**Comment by dolanmiu** [comment]
@samme I can confirm that c975bc9 fixes it

**Comment by dolanmiu** [comment]
When will 2.8.3 be released? At the moment, I will use 2.8.1 until then

**Comment by samme** [comment]
I think we can make a new release soon.
